lib/messenger_platform.rb

```diff
@@ -19,8 +19,8 @@ def image(receiver_id, image_url)
     Api.call(:image, receiver_id, image_url)
   end
 
-  def welcome(page_id, text)
-    Api.call(:welcome, page_id, text)
+  def welcome(template_type, *args)
```

Comment:
@MQuy I think it's the same as `payload`. Since we might need more attributes for `button` type. Also I think I forgot to add `MessageButton` class :D. Adding it now.